sysstat_report.py

```diff
@@ -27,7 +27,7 @@
 from typing import IO, Any, Dict, List, Optional, Sequence, Set, Tuple, Union
 
 ReportType = enum.Enum("ReportType", ("DAILY", "WEEKLY", "MONTHLY"))
-SysstatDataType = enum.Enum("SysstatDataType", ("LOAD", "CPU", "MEM", "SWAP", "NET", "SOCKET", "TCP4", "IO"))
+SysstatDataType = enum.Enum("SysstatDataType", ("LOAD", "CPU", "MEM", "SWAP", "NET", "SOCKET", "TCP4", "IO", "DISK"))
```

Comment:
I suggest renaming `DISK` to `FS_USAGE` to avoid any confusion with `IO`. Also it may not be an actual disk but a SSD, tmpfs, etc.